go/cmd/dolt/commands/diff.go

```diff
@@ -779,26 +1070,74 @@ func diffDoltSchemasTable(
 	return nil
 }
 
+// ArePrimaryKeySetsDiffable checks if two schemas are diffable. Assumes the
```

Comment:
lowercase "A" to match function name

I also don't understand what `if __DOLT__` means. Are you talking about the storage format?

Reply:
Updating the casing.
The comment was taken from a different version of this function and is no longer relevant. Removing comment.